README.md

```diff
@@ -8,17 +8,28 @@ ES6 and Webflow.
 
     yarn run import-webflow zipfile.zip
 
-
 ## Development environment
 
 1. Install dependencies by running `yarn`
-2. Build the static assets by running `yarn build`
-3. Start a development server by running `yarn serve`
+2. Start a development server by running `yarn dev`
+
+The above does not include the service worker so, if you are working on a piece of functionality that requires the service worker, the workflow is a bit different.
+
+With the dependencies installed, ensure that you have a `.env` file in the root of your project with the following entry:
+
+```
+NODE_ENV="production"
+```
 
-`parcel-plugin-sw-cache` does not support hot reloading so you have to `yarn build` each time you want to see code changes reflected in the locally-served site.
+Now run the following:
 
+```
+yarn start
+```
```

Comment:
`parcel build` sets NODE_ENV=production for you, so all you need for service worker is `yarn start`

https://parceljs.org/cli.html#build

Reply:
Great stuff, one less thing to remember.